server/src/main/java/org/cloudfoundry/identity/uaa/logging/SanitizedLogFactory.java

```diff
@@ -6,7 +6,7 @@
 /**
  * Returns Log instance that replaces \n, \r, \t with a | to prevent log forging.
  */
-@SuppressWarnings({"javasecurity:S5145", "javasecurity:S2629"}) // sanitize log messages
+@SuppressWarnings({"javasecurity:S5145", "java:S2629"}) // sanitize log messages
```

Comment:
suppress is sometimes an option, but for security findings not

So I would like to understand first the issue from Sonar (btw. CodeQL has similar findings)

Reply:
https://sonarcloud.io/organizations/cloudfoundry-1/rules?open=javasecurity%3AS5145&rule_key=javasecurity%3AS5145
"Logging should not be vulnerable to injection attacks"
False positive, because we sanitize log messages